patches/002-dcheck.patch

```diff
@@ -36,32 +36,17 @@ These files have debug checks explicitly commented out:
   ui/latency/latency_histogram_macros.h
 
 
-diff --git a/base/BUILD.gn b/base/BUILD.gn
```

Comment:
Before merging, please update the list in the above patch description by removing `base:base` from it.